app/src/main/java/org/jellyfin/androidtv/data/eventhandling/TvApiEventListener.java

```diff
@@ -140,13 +140,17 @@ public void onBrowseCommand(ApiClient client, BrowseRequest command) {
                 Timber.i("Command ignored due to no activity or playback in progress");
                 return;
             }
-            client.GetItemAsync(command.getItemId(), TvApp.getApplication().getCurrentUser().getId(), new Response<BaseItemDto>() {
-                @Override
-                public void onResponse(BaseItemDto response) {
-                    //Create a rowItem and pass to our handler
-                    ItemLauncher.launch(new BaseRowItem(0, response), null, -1, TvApp.getApplication().getCurrentActivity(), true);
-                }
-            });
+            try {
+                client.GetItemAsync(command.getItemId(), TvApp.getApplication().getCurrentUser().getId(), new Response<BaseItemDto>() {
```

Comment:
Why not check if `getItemId()` is null/empty and return instead?

Reply:
yeah that's much simpler